ios/FluentUI/Avatar/AvatarTokens.swift

```diff
@@ -6,179 +6,274 @@
 import UIKit
 import SwiftUI
 
-/// Pre-defined styles of the avatar
-@objc public enum MSFAvatarStyle: Int, CaseIterable {
-    case `default`
-    case accent
-    case group
-    case outlined
-    case outlinedPrimary
-    case overflow
-}
+/// Design token set for the `Avatar` control.
+open class AvatarTokens: ControlTokens {
+    // MARK: - Design Tokens
 
-/// Pre-defined sizes of the avatar
-@objc public enum MSFAvatarSize: Int, CaseIterable {
-    case xsmall
-    case small
-    case medium
-    case large
-    case xlarge
-    case xxlarge
+    /// The size of the content of the `Avatar`.
+    open var avatarSize: CGFloat {
+        return size.size
+    }
 
-    var size: CGFloat {
-        switch self {
+    /// The radius of the corners of the `Avatar`.
+    open var borderRadius: CGFloat {
+        switch style {
+        case .default, .accent, .outlined, .outlinedPrimary, .overflow:
+            return globalTokens.spacing[.none]
+        case .group:
+            switch size {
+            case .xsmall:
+                return globalTokens.borderRadius[.small]
+            case .small, .medium:
+                return globalTokens.borderRadius[.medium]
+            case .large, .xlarge:
+                return globalTokens.borderRadius[.large]
+            case .xxlarge:
+                return globalTokens.borderRadius[.xLarge]
+            }
+        }
+    }
+
+    /// The font used for text in the `Avatar`
+    // We seem to have some odd font choices
+    open var textFont: FontInfo {
+        switch size {
         case .xsmall:
-            return FluentUIThemeManager.S.MSFAvatarTokens.size.xSmall
+            return .init(size: 9, weight: globalTokens.fontWeight[.regular])
```

Comment:
A few of these fonts don't really line up with what we have available in typography.

Reply:
Checked with Figma and that does seem to be the case, yep.